diagnostics.md

```diff
@@ -75,7 +75,12 @@ application container that can be cleaned up (runaway logs, too much local data,
 
 #### test_write_latency
 ##### Summary
-This test compares each partition's average write latency to a predefined target.
+This test compares each partition's average write latency to a predefined target (1s). There are some caveats to this
+test that are worth considering. Since it attempts to categorize a distribution with a point sample, the reported sample
+size should always be considered. Smaller sample sizes are prone to fluctuations that do not necessarily indicate
```

Comment:
Is it possible to give some guidance about what a small/large sample looks like?  Something like "If the sample size is in the [dozens|hundreds|thousands|millions], it's likely to be representative of overall performance; if it's lower than that, it's less trustworthy."

Also, I take it there's no easy way to increase that sample size -- that the number of writes is mainly going to be determined by how often the application writes.  If that's the case, it would be good to make that clear.

Reply:
> Is it possible to give some guidance about what a small/large sample looks like? Something like "If the sample size is in the [dozens|hundreds|thousands|millions], it's likely to be representative of overall performance; if it's lower than that, it's less trustworthy."

I'm open to suggestions on this, but generally I think it's a hard line to draw without generating over/under confidence. More is better, obviously, but I don't want results necessarily discarded just because we're close but under the threshold we set.

> Also, I take it there's no easy way to increase that sample size -- that the number of writes is mainly going to be determined by how often the application writes. If that's the case, it would be good to make that clear.

If you have a potentially flaky card deployed remotely, a write-heavy workload to get more samples is not going to be a great plan :P. I will definitely add something about where these writes come from, though!